### ISSUE TITLE
Automated accessibility test failure for tooltips in story for TranscribedLines

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#1705]
**Package** lib-classifier

**Describe the bug**
The TranscribedLines component uses tooltips where the aria id is dynamically added onto the element when the associated DOM event for focus or hover is triggered and thus is failing automated accessibility tests because the id is missing from the initial HTML markup. Noted in https://github.com/zooniverse/front-end-monorepo/pull/1686#issuecomment-656597385

**To Reproduce**
Use the `TranscribedLines` story and accessibility audit tool.

**Expected behavior**

Accessibility audit test should pass.

**Additional context**
The description is still read out as expected when using VoiceOver. The failure is just in the HTML / automated tested.

**Comment by eatyourgreens** [comment]
I think where this fails is when reading the list of buttons to get an initial overview of the page in the VoiceOver rotor. A sighted reader can see all the lines, which are coloured to indicate status. A VoiceOver user must focus  each line and read them individually in order to hear whether a line is complete or not.

**Comment by eatyourgreens** [comment]
This is blocking tests for the `TranscribedLines` component (#4247.)

**Comment by goplayoutside3** [comment]
Noting that the mentioned automated accessibility test is Storybook. The [TranscribedLines](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/drawing-tools-transcribedlines--default) story does not display lines anymore 🤔 However, there are no accessibility warnings in the story for [Transcription Line](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/drawing-tools-transcription-line--complete) mark.

**Comment by eatyourgreens** [comment]
>The [TranscribedLines](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/drawing-tools-transcribedlines--default) story does not display lines anymore 🤔

I've fixed that in #5822. The transcription task now requires an `enableInteractionLayer` prop to be set in order to display drawn lines.

**Comment by eatyourgreens** [comment]
The accessibility failure here is that `aria-describedby` for each line points to an ID that doesn't exist in the document. In this case `transcribed-0`.

<img width="600" alt="Screenshot of the DOM inspector for an SVG transcribed line, showing the SVG attributes for the SVG group that contains the line." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/7974ee28-f3e0-43ab-b108-e1de76d8dfe5">

**Comment by eatyourgreens** [comment]
To reproduce this:
1. Open the rotor.
2. Navigate to form controls. 
3. Navigate the list of buttons on the page. 
4. Each transcribed line is listed as a button, but the status isn’t announced, because the tooltips aren’t present in the DOM. 

I was too polite about this when it first came up, saying ‘this might be broken’ when it’s totally inaccessible. 

Anytime you see tooltips in a web UI, you should be thinking, ‘this fails a WCAG audit.’

**Comment by eatyourgreens** [comment]
Steve Faulkner has [a fix for accessible names vanishing](https://html5accessibility.com/stuff/2024/09/21/drugs-button-popover/) when a tooltip is hidden, now that `popover` and `popovertarget` are available in browsers.
